packages/manager/modules/netapp/src/order/translations/Messages_fr_FR.json

```diff
@@ -4,7 +4,7 @@
   "netapp_order_performance_level": "Niveau de performance",
   "netapp_order_performance_level_description": "Choisissez le niveau de performance du service selon vos besoins",
   "netapp_order_performance_level_type_Premium": "Licence Premium",
-  "netapp_order_performance_level_type_Premium_description": "Équilibre entre prix et performance.",
+  "netapp_order_performance_level_type_Premium_description": "Équilibre entre prix et performance.\n64Mo/s et 4000 IOPS par To provisionné",
```

Comment:
**nitpick:** Split the sentence and add 2 HTML elements in HTML to avoid breakline char in Trans.